components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml

```diff
@@ -3533,6 +3533,9 @@ components:
         isDefaultVersion:
           type: boolean
           example: false
+        isAWSAPI:
+          type: boolean
+          example: false
```

Comment:
These are not needed I think

Reply:
@malinthaprasan you are proposing that this be changed to something generic like below?

```
vendorType:
    type: string
    default: WSO2
    enum: [WSO2, AWS]
```
In the case vendorType is AWS we can execute AWS API specific logic